wallet/main.cpp

```diff
@@ -1606,13 +1579,6 @@ string GetWarnings(string strFor)
         strStatusBar = strMiscWarning;
     }
 
-    // if detected invalid checkpoint enter safe mode
-    if (Checkpoints::hashInvalidCheckpoint != 0) {
-        nPriority    = 3000;
-        strStatusBar = strRPC = _("WARNING: Invalid checkpoint found! Displayed transactions may not be "
-                                  "correct! You may need to upgrade, or notify developers.");
-    }
-
```

Comment:
Did this alert only apply to SyncCheckpoints?

Reply:
I believe so. It's very badly implemented that nothing is clear. More importantly, it's just a warning.